src/spaces.hpp

```diff
@@ -168,6 +182,11 @@ struct entity<derived<T, N>> {
         : data{std::forward<Args>(xs)...}
     {}
 
+    explicit constexpr entity(std::initializer_list<T> list)  // clang-format on
+    {
+        std::copy(list.begin(), list.end(), data.begin());
```

Comment:
Isn't this problematic if the initializer list is not the right size? You should be able to decay to C-array.

Reply:
Ah, so calling it with an initializer list but having a c array of the right size as argument? Let me try